pg_search/src/postgres/customscan/pdbscan/projections/snippet.rs

```diff
@@ -143,3 +171,66 @@ pub unsafe fn uses_snippets(
     walker(node, addr_of_mut!(context).cast());
     context.snippet_info
 }
+
+pub unsafe fn uses_snippet_positions(
```

Comment:
Rather than continuing to add recursive walks like this to find individual symbols, it seems like we might want to merge them into a single walk to find all of our symbols. But maybe not right here if you're trying to reduce churn.